### PR TITLE
[WFCORE-4127] Add documentation how to disable CLI paging with an option

### PR DESCRIPTION
**Pull request by MMarus**
WFCORE issue: https://issues.jboss.org/browse/WFCORE-4127
EAP issue: https://issues.jboss.org/browse/EAP7-1113
Analysis document: https://github.com/wildfly/wildfly-proposals/pull/131
WFCORE PR: https://github.com/wildfly/wildfly-core/pull/3550

Disable CLI paging with an option through the jboss-cli.xml or a command-line argument.

### CODE REVIEW

File: docs/src/main/asciidoc/_admin-guide/Command_Line_Interface.adoc [hunk 1]
```diff
@@ -576,6 +576,11 @@ You can use up/down arrows to retrieve previously typed text. NB: search history
 * 'N' to jump to the previous match if any. If no search text has been typed,
  then the last entry from the search history is used.
 
+There are two possible ways *to disable* the output paging and write the whole output of the commands at once:
+
+* _--no-output-paging_ will disable output paging;
```
Comment (jfdenise) [inline docs/src/main/asciidoc/_admin-guide/Command_Line_Interface.adoc]: I would say:
_--no-output-paging_ command line option will disable output paging;

File: docs/src/main/asciidoc/_admin-guide/Command_Line_Interface.adoc [hunk 2]
```diff
@@ -576,6 +576,11 @@ You can use up/down arrows to retrieve previously typed text. NB: search history
 * 'N' to jump to the previous match if any. If no search text has been typed,
  then the last entry from the search history is used.
 
+There are two possible ways *to disable* the output paging and write the whole output of the commands at once:
+
+* _--no-output-paging_ command line parameter will disable the output paging;
+* Add _<output-paging>false<output-paging>_ in `jboss-cli.xml`.
```
Comment (MMarus) [inline docs/src/main/asciidoc/_admin-guide/Command_Line_Interface.adoc]: @jfdenise thank you very much for very quick review. I have incorporated the requested changes. WDYT about it?

**Review by jfdenise — APPROVED** [review]
Thank-you!

**Comment by bstansberry** [comment]
Oops; I was too quick. I for no good reason thought this was a doc change for an already completed feature.

@jfdenise I can revert if you prefer and then this can be resubmitted when the rest is ready. I don't mind leaving it though, so long as the feature isn't far away.

**Comment by jfdenise** [comment]
@bstansberry, the feature is implemented, test plan done, need to dev test. I feel that we can keep it, @mnovak1 any concern?

**Comment by mnovak1** [comment]
@jfdenise No concerns. I'm on conference till Thursday but I'll finish test development until the end of this week.

**Pull request merged**